samples/snippets/cpp/VS_Snippets_CLR/console.writelineFmt1/cpp/wl.cpp

```diff
@@ -1,3 +1,4 @@
+// <Snippet1>
```

Comment:
you don't really need this. you can just refer to the snippet without specifying a region.